internal/pkg/cli/task_run.go

```diff
@@ -523,7 +526,8 @@ func (o *runTaskOpts) askTaskGroupName() error {
 		taskRunGroupNamePrompt,
 		taskRunGroupNamePromptHelp,
 		basicNameValidation,
-		prompt.WithFinalMessage("Task group name:"))
+		prompt.WithFinalMessage("Task group name:"),
```

Comment:
Should we even prompt for the group name if it's optional? I think we can skip it and users can enter a separate name through flags

Reply:
Is it possible that they accidentally forget to (or don't know they can) use that flag, and everything that's on the stack get updated unexpectedly?

Reply:
Gotcha, yeah that makes sense. Then let's just remove the "Optional." word from the description of the flag but have the default there so they can just press enter

Reply:
A different default that Docker Compose does I believe is the name of the directory: https://docs.docker.com/compose/reference/overview/ (see --project) which might be interesting